go/src/koding/klientctl/ctlcli/ctlcli.go

```diff
@@ -116,9 +116,10 @@ func ExitErrAction(f ExitingErrCommand, log logging.Logger, cmdName string) cli.
 		}
 
 		Close()
-		ExitFunc(exit)
```

Comment:
Hm, i need the timing per command, could we generalize and exit with 1 if there is an error till we migrate to cobra?